raiden_contracts/contract_source_manager.py

```diff
@@ -55,6 +55,7 @@ def relativise(path):
                     output_values=PRECOMPILED_DATA_FIELDS + ['ast'],
                     import_remappings=import_dir_map,
                     optimize=False,
+                    evm_version='byzantium',
```

Comment:
I think `evm_version` should be recorded in `contracts.json`.

Reply:
How do I do this?

Reply:
* change `contract_manager.py` to read/write the new field
* set up a default value for `evm_version`.

Maybe I should do it later.   When I modify `contract_manager.py`,  I can also manually add the field to the existing `contracts.json`s with the new solc versions.